src/screens/DiscoveryScreenV2/fragment/TopicFragment.js

```diff
@@ -186,8 +143,8 @@ const TopicFragment = ({
         {renderRecentSearch(index)}
         <View style={styles.domainContainer}>
           <DomainList
-            handleSetFollow={() => handleFollow(from, true, item, index)}
-            handleSetUnFollow={() => handleFollow(from, false, item, index)}
+            handleSetFollow={() => handleFollow(from, true, item)}
+            handleSetUnFollow={() => handleFollow(from, false, item)}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made here seem to be an improvement in terms of code readability and maintainability. The removal of the `index` parameter from the `handleFollow` and `handleSetUnFollow` functions simplifies their usage. However, it's important to ensure that these functions handle all edge cases and perform as expected.


<!-- This is an auto-generated comment by OSS CodeRabbit -->